modules/ROOT/pages/usage-reports.adoc

```diff
@@ -1,23 +1,24 @@
-= Viewing Usage Reports
+= Usage Reports
 :page-aliases: gateway::flex-view-usage-reports.adoc
 
-Usage reports provide insights into your monthly usage of Anypoint Platform products. 
+Usage reports provide comprehensive monthly and daily summaries that give you valuable insights into your resource use across Anypoint Platform products. By correlating your usage with billing cycles, you can gain insights into the factors influencing your costs, which enables you to proactively manage your budget and make informed decisions to optimize your resource allocation.
 
-When you use an Anypoint Platform product, MuleSoft meters your usage and aggregates the records on a monthly basis. Usage reports are updated on the third day of each month and are split by product. All reports are calculated using the UTC time standard.
+When you use an Anypoint Platform product that is metered, MuleSoft measures your usage and aggregates the records on a daily and monthly basis. Monthly and daily usage reports have a three-day latency, meaning that today’s usage is available in three days. All reports are calculated using the UTC time standard. For example, a daily usage is aggregated from 00:00 to 00:00 UTC. 
 
-Use the usage report to compare your actual usage against your existing contract. The report is not for billing purposes. 
+Daily usage retention is three months and monthly usage retention is up to five years.
+
+Organizations that have a usage-based pricing model can compare actual usage from the report against the existing contract. However, usage reports are not for billing purposes. For more information about usage metrics and pricing, read:
 
 * xref:pricing.adoc[]
-* xref:pricing-metrics.adoc[]
+* xref:usage-metrics.adoc[]
 
```

Comment:
Is the formatting correct here? Should this be one big bullet list? If not, then maybe the xref shouldn't be a bullet.